contrib/depends/packages/freebsd_base.mk

```
@@ -1,6 +1,6 @@
 package=freebsd_base
 $(package)_version=11.3
-$(package)_download_path=https://download.freebsd.org/ftp/releases/amd64/$($(package)_version)-RELEASE/
+$(package)_download_path=https://archive.freebsd.org/old-releases/amd64/$($(package)_version)-RELEASE/
```

```suggestion
$(package)_download_path=https://download.freebsd.org/releases/amd64/$($(package)_version)-RELEASE/
```
Same (for 14.1)

note.. no need to change. The `archive` link has old _and_ current files.. but the `download` link is keeping the existing link.

---

thankfully, updating to 14.1 breaks _everything_ so we don't have to worry about this in this PR. adding a working mirror for the current version should be fine